Makefile

```diff
@@ -1,4 +1,5 @@
 ipynbs/%.ipynb: %.jl
+	julia --project scripts/setup.jl
```

Comment:
This line enables the automatic execution of `Pkg.instantiate()`. Therefore, after cloning the git repository, we need only to execute the `make` command.